Zend/tests/enum/traits-no-properties.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+Enum cannot have properties, even via traits
+--FILE--
+<?php
+
+trait Rectangle {
+    protected string $shape = "Rectangle";
+
+    public function shape(): string {
+        return $this->shape;
+    }
+}
+
+enum Suit {
+    use Rectangle;
+
+    case Hearts;
+    case Diamonds;
+    case Clubs;
+    case Spades;
+}
+
+var_dump((new ReflectionClass(Suit::class))->getProperties());
```

Comment:
nit: Split this out into a separate test of a valid enum - (optional $value) and $name are the expected properties of ReflectionClass->getProperties() and the throwing isn't caused by getProperties

```suggestion
var_dump(Suit::Hearts);
```

Reply:
I removed the var_dump completely to make sure this gets triggered even when just declaring the class.